app/assets/stylesheets/topology.css

```diff
@@ -79,6 +79,10 @@ _:-ms-lang(x), _:-webkit-full-screen, kubernetes-topology-icon svg /* Edge hack
   stroke: #bbb;
 }
 
+.kube-topology g circle.information {
+  stroke: #00c;
+}
+
```

Comment:
It doesn't need to be here. But the case statement selecting which css class will be rendered is in the general topology_service.js. I can move it to the middleware css and place a note in topology js stating where that style is defined and if somebody else needs it, she/he can move the style to the general topology css.